native/cocos/scene/RenderScene.cpp

```diff
@@ -109,6 +109,7 @@ void RenderScene::removeCamera(Camera *camera) {
 void RenderScene::removeCameras() {
     for (const auto &camera : _cameras) {
         camera->detachFromScene();
+        camera->destroy();
```

Comment:
Why adding `destroy` here? It will be different logic with web.

Reply:
Yep, should not invoke destroy here. A camera can be attached again or be attached to other scene.